src/mars.d

```diff
@@ -951,6 +951,11 @@ Language changes listed by -transition=id:
                     global.params.objname = p;
                     continue;
                 }
+                if (strcmp(p, `/?`) == 0)
```

Comment:
Not sure important as having a file in unix systems called `/?` is almost unthinkable, but it would be nice if this only works in windows, the only platform where it makes sense.